### ISSUE TITLE
Button Layout

### ISSUE THREAD

**Issue by LouisJS** [react-native-training/react-native-elements#842]
### Description

1. Explain what you did

I'm implementing a simple View with Buttons at the top

2. What you expected to happen

Having styled Buttons sticked to the top of the View

3. What actually happens

https://res.cloudinary.com/donglyhya/image/upload/v1516875715/Screenshot_20180125-110116_hk3ixr.png

**"react-native-elements": "^1.0.0-beta",**

### Reproduction Steps and Sample Code

```render() {
    return (
      <View style={{flex: 1, paddingTop: 20,}}>
          <Button
            text='Log out'
            loading={false}
            loadingProps={{ size: 'small', color: 'white' }}
            buttonStyle={{ height: 50, width: 230, backgroundColor: 'rgba(111, 202, 186, 1)', borderRadius: 5 }}
            textStyle={{ fontWeight: 'bold', fontSize: 23 }}
            onPress={() => console.log('hello')}
            underlayColor="transparent"
          />
          <Button
            text='Log out'
            loading={false}
            loadingProps={{ size: 'small', color: 'white' }}
            buttonStyle={{ height: 50, width: 230, backgroundColor: 'rgba(111, 202, 186, 1)', borderRadius: 5 }}
            textStyle={{ fontWeight: 'bold', fontSize: 23 }}
            containerStyle={{ marginVertical: 10 }}
            onPress={() => console.log('aye')}
            underlayColor="transparent"
          />
      </View>
    );
  }
```

**Comment by xavier-villelegier** [comment]
Mehh, good catch @LouisJS. Thanks for your recent activity on this repo, much appreciated 🙏

@Monte9, there is a `flex: 1` in the button container style [here](https://github.com/react-native-training/react-native-elements/blob/next/src/buttons/Button.js#L131), which was not the case before the refactoring

**Comment by LouisJS** [comment]
You're welcome ! i really enjoy working with this library ! 😄 

I've noticed when you start an snack expo project, it uses the `0.19.0` versions of react-native-elements.

Is it possible to use `^1.0.0-beta` ? That would be easier to report issues concerning the beta.

**Comment by iRoachie** [comment]
This would actually be great to have, not sure if there’s a way to specify for snack to use a specific version. cc @brentvatne

**Comment by xavier-villelegier** [comment]
@iRoachie @LouisJS Actually, you can specify the version like that

```
import { Button } from 'react-native-elements'; // 1.0.0-beta
```

Have a look at [this snack](https://snack.expo.io/SkUwcDPrz) running the example of @LouisJS with the `v1.0.0-beta`

**Comment by iRoachie** [comment]
Oh darn, sweet! We should probably document this somewhere in the readme. Pretty useful for when users want to show a bug in a particular version

**Comment by iRoachie** [comment]
Fixed this as a part of https://github.com/react-native-training/react-native-elements/commit/e0a4b6afda185c215fa9ad9fbfbc7f324c73cbbe#diff-a0a6368880417eca5ad26c484e9ded7aL131. Will be available in next beta release.

**Comment by Amurmurmur** [comment]
Why button component now depends on expo component?
Is it an absolute need from now on to use expo?

**Comment by xavier-villelegier** [comment]
@Amurmurmur This will be partially fixed in the next beta, please have a look at #826

**Comment by xavier-villelegier** [comment]
Fixed in `v1.0.0-beta2` 👍
